README.md

```diff
@@ -66,6 +66,37 @@ of your `pom.xml` and the `dagger-compiler` artifact as either an `optional` or
 </dependencies>
 ```
 
+or as an `annotationProcessorPaths` value of the `maven-compiler-plugin`
+(requires at least version 3.5):
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>com.google.dagger</groupId>
+    <artifactId>dagger</artifactId>
+    <version>2.0.2</version>
```

Comment:
Can you leave this as `2.x` so we don't need to constantly update it? We updated this after you sent out this PR.